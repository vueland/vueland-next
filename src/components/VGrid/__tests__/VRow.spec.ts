import { mount } from '@vue/test-utils'
import { VRow } from '../'

describe('VRow', () => {
  const mountFunction = (options = {}) =>
    mount(VRow, {
      ...options,
    })

  it('should test "align-start" prop and match snapshot', () => {
    const wrapper = mountFunction({
      props: {
        alignXl: 'center',
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('should test "justify-center" props and match snapshot', () => {
    const wrapper = mountFunction({
      props: {
        justifyXl: 'start',
      },
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
