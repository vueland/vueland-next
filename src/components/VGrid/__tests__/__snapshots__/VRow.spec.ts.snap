// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VRow should test "align-start" prop and match snapshot 1`] = `
<div class="v-row align-xl-center">
</div>
`;

exports[`VRow should test "justify-center" props and match snapshot 1`] = `
<div class="v-row justify-xl-start">
</div>
`;
